man-roxygen/roxlate-ml-tol.R

```diff
@@ -0,0 +1 @@
+#' @param tolerance Param for the convergence tolerance for iterative algorithms..
```

Comment:
Removed :) in https://github.com/rstudio/sparklyr/pull/179/commits/e0321c7c20d207cab468914588a624bc4225873d